common-theme/layouts/partials/block/link.html

```diff
@@ -0,0 +1,20 @@
+{{ $blockData := .Page.Scratch.Get "blockData" }}
+{{ $randomImage := (index (resources.Match "custom-images/backgrounds/**.**" | shuffle | first 1) 0) }}
+{{ $randomImage = $randomImage.Resize "480x" }}
+
+
+<section class="c-block c-block--{{ $blockData.type }}">
+  <a
+    id="{{ $blockData.name |urlize }}"
+    href="{{ $blockData.sot }}"
```

Comment:
What does `sot` stand for? Should this be `src`?

Reply:
source of truth - sometimes you need the link and then a transformed link to say an api

{{/* Set the source of truth and api defaults to src */}}
{{ .Scratch.SetInMap "blockData" "sot" $src }}
{{ .Scratch.SetInMap "blockData" "api" $src }}

Reply:
Oh interesting - I replied via the VSCode integration I just set up (it's what the trainees use) and it didn't add to the thread, just added a new comment.

Replying directly in VSCode seems like something we should figure out because it's something volunteers identify repeatedly about the problems with code review (getting demotivated because trainees ignore them)